code/modules/antagonists/revenant/revenant.dm

```diff
@@ -37,7 +37,7 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	maxbodytemp = INFINITY
-	harm_intent_damage = 0
+	harm_intent_damage = 0.00001
```

Comment:
What is the reason for this?

Reply:
If harm intent damage has no value, it defaults to the attacker'a punchdamage. This makes it as close to 0 as possible without being 0

Reply:
This seems like a flawed system if you need a workaround like this

Reply:
revenants are really fucking weird and the exception shouldnt influence the rule